docs/development-guide/EFMAndFailoverPluginPerformanceResults.md

```diff
@@ -32,14 +32,18 @@
 | releaseResourcesWithNoPlugins                | 8.030   | 3.776  | us/op |
```

Comment:
Can you change the file name to PluginPerformanceResults? You'll also have to search for references to this file in the documentation and change the references